corehq/apps/groups/models.py

```diff
@@ -222,7 +235,7 @@ def get_reporting_groups(cls, domain):
             startkey=key,
             endkey=key + [{}],
             include_docs=True,
-            # stale=settings.COUCH_STALE_QUERY,
+            stale=settings.COUCH_STALE_QUERY,
```

Comment:
Ok, I just checked and they definitely all will be fine with stale=ok, so this is an improvement!